packages/core/src/scss/component/accordion/_defaults.scss

```diff
@@ -48,6 +51,22 @@ $accordion-item-toggle-disabled-transform: $accordion-item-toggle-initial-transf
 $accordion-item-toggle-pseudo-initial-transform: none !default;
 $accordion-item-toggle-pseudo-final-transform: rotate(-180deg);
 
+// Expand transform properties.
+$accordion-expand-transform: none !default;
+$accordion-expand-visited-transform: $accordion-expand-transform !default;
+$accordion-expand-focus-transform: $accordion-expand-transform !default;
+$accordion-expand-hover-transform: $accordion-expand-transform !default;
+$accordion-expand-active-transform: $accordion-expand-transform !default;
+$accordion-expand-disabled-transform: $accordion-expand-transform !default;
+
+// Collapse transform properties.
+$accordion-collapse-transform: none !default;
+$accordion-collapse-visited-transform: $accordion-collapse-transform !default;
+$accordion-collapse-focus-transform: $accordion-collapse-transform !default;
+$accordion-collapse-hover-transform: $accordion-collapse-transform !default;
+$accordion-collapse-active-transform: $accordion-collapse-transform !default;
+$accordion-collapse-disabled-transform: $accordion-collapse-transform !default;
+
```

Comment:
The way transforms are handled have changed.

You can see above in the file where we have "initial", "final" and "disabled" transforms, but nothing else.

You'll need to adjust to match.